include/Core/Users/UsersModule.php

```diff
@@ -87,7 +89,12 @@ protected function delete_users_from_query( $query, $options ) {
 				continue;
 			}
 
-			$deleted = wp_delete_user( $user->ID );
+			if ( isset( $options['reassign_user'] ) ) {
+				$deleted = $options['reassign_user'] ? wp_delete_user( $user->ID, $options['reassign_user_id'] ) : wp_delete_user( $user->ID );
```

Comment:
Remove the ternary operator and move the non empty check to the above if condition.